mathics/builtin/calculus.py

```diff
@@ -534,9 +534,9 @@ class Root(Builtin):
     """
 
     messages = {
-        'nuni': "`1` is not a univariate polynomial",
-        'nint': "`1` is not an integer",
-        'iidx': "`1` is out of bounds"
+        'nuni': "Argument `1` at position 1 is not a univariate polymonial",
```

Comment:
There's a typo in here

Reply:
> @weakit @sn6uv @bnjones I'd like to get your input on this.

I'm not really familiar with how mathics works, and I'm not that good at working with big projects. I can help with bugs, but that's the most I think I'll be able to do.

Reply:
Fixed in https://github.com/mathics/Mathics/pull/806/commits/9c53aed4bd0de82d97d2e2f944927d4e3ab10105